common/persistence/nosql/nosqlplugin/cassandra/workflow.go

```diff
@@ -725,10 +728,10 @@ func (db *cdb) InsertReplicationTask(ctx context.Context, tasks []*nosqlplugin.R
 	shardID := shardCondition.ShardID
 	batch := db.session.NewBatch(gocql.LoggedBatch).WithContext(ctx)
 	for _, task := range tasks {
-		createReplicationTasks(batch, shardID, task.DomainID, task.WorkflowID, []*nosqlplugin.ReplicationTask{task})
+		createReplicationTasks(batch, shardID, task.DomainID, task.WorkflowID, []*nosqlplugin.ReplicationTask{task}, time.Now())
```

Comment:
is the db.timesource available for these too?

Reply:
Yes. Update it. 
But why is db.timesource.now() better than time.Now() ?